recipes-debian/libnss-mdns/libnss-mdns_debian.bb

```diff
@@ -0,0 +1,47 @@
+# base recipe: meta/recipes-support/libnss-mdns/libnss-mdns_0.10.bb
+# base branch: warrior
+# base commit: 38d5c8ea98cfa49825c473eba8984c12edf062be
+
+SUMMARY = "Name Service Switch module for Multicast DNS (zeroconf) name resolution"
+HOMEPAGE = "http://0pointer.de/lennart/projects/nss-mdns/"
+SECTION = "libs"
+
+LICENSE = "LGPLv2.1+"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2d5025d4aa3495befef8f17206a5b0a1"
+
+DEPENDS = "avahi"
```

Comment:
@yuwata Does debian source have ``` --enable-avahi ``` option in configure script?  If so, it would be nice to set  ``` --enable-option ``` to EXTRA_OECONF to have same feature that poky provides. If not , you can remove DEPENDS line.

Reply:
The option was removed in v0.11. See https://github.com/lathiat/nss-mdns/commit/8f731b79df00686c95bc58e42d8bb1c989e81c71.

Reply:
I see!